### DISCUSSION TITLE
OOP-IEC61131-3--Curso-Youtube/en/Concepts%20Previous/Tipos%20de%20datos/

### DISCUSSION THREAD

**Discussion by giscus** [runtimevic/OOP-IEC61131-3--Curso-Youtube#18] [General]
# OOP-IEC61131-3--Curso-Youtube/en/Concepts%20Previous/Tipos%20de%20datos/

IEC 61131-3 PLC OOP Course

https://runtimevic.github.io/OOP-IEC61131-3--Curso-Youtube/en/Concepts%20Previous/Tipos%20de%20datos/

**Comment by Kgbqiang** [comment]
您好，为什么我看不到内容

**Reply by runtimevic** [reply]
> 您好，为什么我看不到内容

Hello @Kgbqiang 
only spanish, If you want you can contribute to translate it, make a pull request...
